tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -0,0 +1,231 @@
+
+	test( 'Title alignment setting', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			container: true,
+			'nested-elements': true,
+		} );
+
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Add widgets.
+		await editor.addWidget( 'nested-tabs', container );
+		await editor.getPreviewFrame().waitForSelector( '.e-n-tabs-content .e-con.e-active' );
+
+		// Act.
+		// Set tabs direction to 'stretch'.
+		await page.locator( '.elementor-control-tabs_justify_horizontal .elementor-control-input-wrapper .eicon-h-align-stretch' ).click();
+		// Set align title to 'start'.
+		await page.locator( '.elementor-control-title_alignment .elementor-control-input-wrapper .eicon-text-align-left' ).click();
+
+		// Assert.
+		// Check if title's are aligned on the left.
+		await expect( editor.getPreviewFrame().locator( '.elementor-widget-n-tabs .e-n-tabs-heading .e-n-tab-title.e-active' ) ).toHaveCSS( 'justify-content', 'flex-start' );
+
+		await wpAdmin.setExperiments( {
+			container: false,
+			'nested-elements': false,
+		} );
+	} );
+} );
+
+test.describe( 'Nested Tabs - Tab icon vertical alignment', async () => {
+	let locationOption;
+	const locationOptions = [ 'Start', 'Center', 'End', 'Justified' ];
+
+	for ( locationOption of locationOptions ) {
+		test( `Should be vertically centered when Justify is set to ${ locationOption }`, async ( { page }, testInfo ) => {
+			const wpAdmin = new WpAdminPage( page, testInfo );
+			await wpAdmin.setExperiments( {
+				container: true,
+				'nested-elements': true,
+			} );
+
+			const editor = await wpAdmin.useElementorCleanPost(),
+				container = await editor.addElement( { elType: 'container' }, 'document' );
+
+			// Add widgets.
+			await editor.addWidget( 'nested-tabs', container );
+			await editor.getPreviewFrame().waitForSelector( '.e-n-tabs-content .e-con.e-active' );
+
+			// Set icons to tabs according 'TabsIcons' array.
+			await setIconsToTabs( page, TabsIcons );
+			const icon = await editor.getPreviewFrame().locator( '.elementor-widget-n-tabs .e-n-tab-title .e-n-tab-icon' ).first(),
+				activeTabIcon = await editor.getPreviewFrame().locator( '.elementor-widget-n-tabs .e-n-tab-title .e-n-tab-icon.e-active' ).first(),
+				currentContext = editor.getPreviewFrame();
+
+			// Assert
+			await page.locator( `.elementor-control-tabs_justify_horizontal [original-title='${ locationOption }']` ).first().click();
+			await expect( icon ).toHaveCSS( 'align-items', 'center' );
+			await clickTab( currentContext, '1' );
+			await expect( activeTabIcon ).toHaveCSS( 'align-items', 'center' );
+			await clickTab( currentContext, '0' );
+
+			await wpAdmin.setExperiments( {
+				container: false,
+				'nested-elements': false,
+			} );
+		} );
+	}
+} );
+
+test.describe( 'Nested Tabs - Inline font icons experiment', async () => {
```

Comment:
Please use one `test.describe()` for all nested tabs tests.

Reply:
We should discuss this.
Not sure that using one `describe` per file is the best practice.